lib/rules/require-early-return.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(options) {
+        assert(
+            options === 'noElse',
```

Comment:
Function to check:
```javascript
function isPercentage(val) {
  if (val >= 0) {
    if (val <= 100) {
      return true;
    } else {
      return false;
    }
  } else {
    return false;
  }
}
```

With `requireEarlyReturn: 'noElse'`:
```javascript
function isPercentage(val) {
  if (val >= 0) {
    if (val <= 100) {
      return true;
    }
    return false;
  }
  return false;
}
```

With `requireEarlyReturn: 'strict' // or true`:
```javascript
function isPercentage(val) {
  if (val < 0) {
    return false;
  }

  if (val > 100) {
    return false;
  }

  return true;
}
```